recipe/activate-clang_win-64.bat

```diff
@@ -10,6 +20,6 @@ set "NM=llvm-nm.exe"
 set "LD=lld-link.exe"
 
 set "CPPFLAGS_USED=-D_CRT_SECURE_NO_WARNINGS -D_MT -D_DLL -nostdlib -Xclang --dependent-lib=msvcrt -fuse-ld=lld -fno-aligned-allocation"
-set "LDFLAGS=-nostdlib -Xclang --dependent-lib=msvcrt -fuse-ld=lld -Wl,-defaultlib:%PREFIX%/lib/clang/@PKG_VERSION@/lib/windows/clang_rt.builtins-x86_64.lib"
+set "LDFLAGS=-nostdlib -Xclang --dependent-lib=msvcrt -fuse-ld=lld -Wl,-defaultlib:%CLANG_BUILD_PREFIX%/lib/clang/@PKG_VERSION@/lib/windows/clang_rt.builtins-x86_64.lib"
```

Comment:
```suggestion
set "LDFLAGS=-nostdlib -Xclang --dependent-lib=msvcrt -fuse-ld=lld -Wl,-defaultlib:%CONDA_PREFIX%/lib/clang/@PKG_VERSION@/lib/windows/clang_rt.builtins-x86_64.lib"
```

Reply:
Is there any documentation about all these prefixes? I would like to understand them more before moving to this again.

Reply:
Ah, figured. This is defined by the order of environment activations; will test and then push.